services/bots/github_apps.py

```diff
@@ -225,18 +245,15 @@ def get_github_app_info_for_owner(
         owner, installation_name, repository
     )
     apps_matching_criteria_count = len(apps_to_consider)
```

Comment:
Does `apps_to_consider` have a hierarchy of which app to try first over another?

Reply:
yes, as documented in the function that produces this list: https://github.com/codecov/worker/blob/8eb5cc4dd98e754c111552eb619b70464df9ea5a/services/bots/github_apps.py#L58

The ordering of apps in the list is the hierarchy.

Reply:
I see I see